src/lib/UserConfig.ts

```diff
@@ -0,0 +1,59 @@
+import fs = require("fs");
+import os = require("os");
+
+const homedir = os.homedir();
+const configModule = `${homedir}/.newsletter-cli/config`;
+
+export type Config = {
+  template_subject?: string
+  template_css?: string
+  template_body?: string
+  source_default?: string
+  ses_send_configuration?: object
+}
+
+export class UserConfig {
+  private _config: Config;
```

Comment:
this should be `unknown` so that TS checks that the type checking is correct.

Reply:
I don't think this is quite right. When I try using unknown there, I get compiler errors:

```
npm run compile 

> newsletter-cli@0.4.0 compile /home/hunmonk/git/newsletter-cli
> npm run clean && tsc -p .


> newsletter-cli@0.4.0 clean /home/hunmonk/git/newsletter-cli
> rm -rf dist

[:~/git/newsletter-cli] custom-config-file(+0/-0) ± npm run compile 

> newsletter-cli@0.4.0 compile /home/hunmonk/git/newsletter-cli
> npm run clean && tsc -p .


> newsletter-cli@0.4.0 clean /home/hunmonk/git/newsletter-cli
> rm -rf dist

src/lib/UserConfig.ts:17:29 - error TS2571: Object is of type 'unknown'.

17     const templateSubject = this._config.template_subject;
                               ~~~~~~~~~~~~

src/lib/UserConfig.ts:24:25 - error TS2571: Object is of type 'unknown'.

24     const templateCss = this._config.template_css;
                           ~~~~~~~~~~~~

src/lib/UserConfig.ts:31:26 - error TS2571: Object is of type 'unknown'.

31     const templateBody = this._config.template_body;
                            ~~~~~~~~~~~~

src/lib/UserConfig.ts:38:27 - error TS2571: Object is of type 'unknown'.

38     const sourceDefault = this._config.source_default;
                             ~~~~~~~~~~~~

src/lib/UserConfig.ts:45:34 - error TS2571: Object is of type 'unknown'.

45     const sesSendConfiguration = this._config.ses_send_configuration;
                                    ~~~~~~~~~~~~


Found 5 errors.
```
I *think* the right approach is to declare `unknown` for all the `type Config` attributes, which seems to compile and work ok, and my latest push includes that fix.

Reply:
alright, thanks!